azurerm/resource_arm_cosmos_db_account.go

```diff
@@ -190,6 +191,21 @@ func resourceArmCosmosDBAccount() *schema.Resource {
 				Set: resourceAzureRMCosmosDBAccountGeoLocationHash,
 			},
 
+			"capabilities": {
```

Comment:
given the way the API's designed, would it make sense for this to match the API design incase additional fields are added in the future?

Reply:
👍